torch/distributed/tensor/_collective_utils.py

```diff
@@ -79,6 +77,8 @@ def mesh_scatter(
     mesh: DeviceMesh,
     mesh_dim: int = 0,
     async_op: bool = False,
+    *,
+    group_src: int = 0,
```

Comment:
do you think the doc string is worth updating, too?

Reply:
done